voice/bxml/verbs/playAudio.md

```diff
@@ -1,7 +1,14 @@
 {% method %}
 ## XML: `<PlayAudio>`
-The PlayAudio verb is used to play an audio file in the call.  The URL of an audio file should be included in the body
-of the `<PlayAudio>` tag.  If a relative URL is given, it is resolved relative to the endpoint that returned the BXML.
+The PlayAudio verb is used to play an audio file in the call.  The URL of an audio file should be
+included in the body of the `<PlayAudio>` tag.  If a relative URL is given, it is resolved relative
+to the endpoint that returned the BXML.
```

Comment:
We will use the same credentials in that case too, right?

Reply:
Yeah. All I did here was tweak spacing though.